### PR TITLE
Pricing Publish

### PR DESCRIPTION
**Pull request by Dann1y**
- free-plan
- plan-list
- feature-list(Mobile, Desktop)
- faq

https://user-images.githubusercontent.com/29726020/111131666-d262c900-85bb-11eb-84ce-ba926494e840.mov

### CODE REVIEW

**Comment by softmarshmallow** [comment]
LGTM

File: web/components/feature-choice/index.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                    </Flex>
+                  )}
+                </Feature>
+              ))}
+            </Text>
+          </Item>
+        ))}
+
+        <Text fontSize="16px" fontWeight="bold" style={{ cursor: "pointer" }}>
+          View all <Icon name="arrowDown" isVerticalMiddle />{" "}
```
Comment (GwonHeeJun) [inline web/components/feature-choice/index.tsx]: delete `{" "}` @Dann1y

**Comment by softmarshmallow** [comment]
image
View all not working

File: web/components/feature-choice/index.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        </Text>
+      </Flex>
+    </Card>
+  );
+};
+
+const Card = styled(Flex)``;
```
Comment (GwonHeeJun) [inline web/components/feature-choice/index.tsx]: What is this? Why didn't you define it and use it?

File: web/components/question/index.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+export default Question;
+
+const Title = styled(Text)`
+  font-size: 20px;
+  letter-spacing: 0em;
+  text-align: left;
+`;
+
+const Wrapper = styled(Flex)``;
```
Comment (GwonHeeJun) [inline web/components/question/index.tsx]: > Why didn't you define it and use it?

too

**Comment by softmarshmallow** [comment]
image
This should be implemented on current `main`

File: web/components/start-now/index.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  }
+`;
+
+const RightWrapper = styled(Flex)`
+  ${props => media("0px", (props.theme as ThemeInterface).breakpoints[0])} {
+    height: 50%;
+  }
+`;
+
+const DescWrapper = styled(Flex)``;
```
Comment (GwonHeeJun) [inline web/components/start-now/index.tsx]: > Why didn't you define it and use it?

too

File: web/pages/pricing/index.tsx [hunk 5]
```diff
@@ -1,21 +1,6 @@
...
-            of payment for now. Feel free to create your awesome products with
-            it.
-          </p>
-        </span>
-      </div>
-    </>
-  );
-}
\ No newline at end of file
+  return <>{PricingSection.map(item => item.content())}</>;
```
Comment (GwonHeeJun) [inline web/pages/pricing/index.tsx]: delete fragment, use like this 

```ts
return PricingSetction.map(item => item.content()}

**Comment by softmarshmallow** [comment]
Start now / start free plan should redirect to URLS.
`open(URLS.landing.signup);`

File: web/sections/pricing/feature-list/desktop-view/index.tsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Flex, Text, Heading } from "rebass";
+import Icon from "components/icon";
+import SectionLayout from "layout/section";
+
+interface FeatureProps {
+  data: any;
```
Comment (GwonHeeJun) [inline web/sections/pricing/feature-list/desktop-view/index.tsx]: Define the type of data.

File: web/sections/pricing/feature-list/mobile-view/index.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import { Flex, Text, Heading } from "rebass";
+import Icon from "components/icon";
+import SectionLayout from "layout/section";
+import FeatureChoice from "components/feature-choice";
+
+interface FeatureProps {
+  data: any;
```
Comment (GwonHeeJun) [inline web/sections/pricing/feature-list/mobile-view/index.tsx:9]: > Define the format of the data.

too

**Comment by softmarshmallow** [comment]
image
image

image


Help context dialog not implemented

File: web/sections/pricing/feature-list/desktop-view/index.tsx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+          mb="30px"
+        >
+          <Heading flex="1" fontSize="56px" color="#000000" fontWeight="bold">
+            Features
+          </Heading>
+          <Flex flex="1">
+            <Content flex="1">Free</Content>
+            <Content flex="1">Team</Content>
+            <Content flex="1" alignItems="center">
+              Extra usage{" "}
```
Comment (GwonHeeJun) [inline web/sections/pricing/feature-list/desktop-view/index.tsx]: delete `{" "}`

**Review by GwonHeeJun — COMMENTED** [review]
Please apply the comment

**Comment by softmarshmallow** [comment]
image


Apologies that never mentioned. this header is sticky on scroll.
When the table hits the main header (appbar), table header shall sticky - visible to scrolling user. the text ("Features") works separately : shall be normal, not sticky.
image

**Comment by softmarshmallow** [comment]
head meta page title to "Pricing - Bridged"
image

**Pull request merged**